playwright-e2e/tests/dsm/medical-record-request.spec.ts

```diff
@@ -0,0 +1,295 @@
+import { expect } from '@playwright/test';
+import { test } from 'fixtures/dsm-fixture';
+import { AdditionalFilter, CustomViewColumns } from 'dsm/component/filters/sections/search/search-enums';
+import { StudyEnum } from 'dsm/component/navigation/enums/selectStudyNav-enum';
+import ParticipantListPage from 'dsm/pages/participant-list-page';
+import { assertDateFormat } from 'utils/test-utils';
+import { logInfo } from 'utils/log-utils';
+import ParticipantPage from 'dsm/pages/participant-page/participant-page';
+import { TabEnum } from 'dsm/component/tabs/enums/tab-enum';
+import MedicalRecordsTab from 'dsm/pages/medical-records/medical-records-tab';
+import { MainInfoEnum } from 'dsm/pages/participant-page/enums/main-info-enum';
+import { SortOrder } from 'dss/component/table';
+import MedicalRecordsRequestPage from 'dsm/pages/medical-records/medical-records-request-page';
+import { FieldsEnum } from 'dsm/pages/medical-records/medical-records-enums';
+import Input from 'dss/component/input';
+import MedicalRecordsTable from 'dsm/pages/medical-records/medical-records-table';
+
+test.describe.serial('Medical records request workflow', () => {
+  let shortId: string;
+  let confirmedInstitution: string;
+
+  const confirmedInstitutions = [
+    'Massachusetts General Hospital',
+    'Asante Three Rivers Medical Center',
+    'Bothwell Regional Health Center',
+    'Adcare Hospital of Worcester Inc',
+    'Brigham and Women\'s',
+    'Asante Three Rivers Medical Center',
+    'Boston Children\'s Hospital'
+  ];
+
+  const studies: StudyEnum[] = [StudyEnum.OSTEO2];
+
+  for (const study of studies) {
+    test(`Update Institution @dsm @${study}`, async ({ page, request }) => {
+      let participantPage: ParticipantPage;
+      let medicalRecordsRequestPage: MedicalRecordsRequestPage;
+      let rowIndex = -1;
+
+      const participantListPage = await ParticipantListPage.goto(page, study, request);
+      const participantListTable = participantListPage.participantListTable;
+      const customizeViewPanel = participantListPage.filters.customizeViewPanel;
+      const searchPanel = participantListPage.filters.searchPanel;
+
+      await test.step('Find an enrolled participant', async () => {
+        const registrationDateColumn = 'Registration Date';
+        // These are columns displayed in Medical Records table
+        const typeColumn = 'Type';
+        const institutionNameColumn = 'Institution Name';
+        const followUpColumn = 'Follow-Up required';
+        const mrReviewColumn = 'MR Review';
+        const mrProblemColumn = 'MR Review';
+        const initialMRRequestColumn = 'Initial MR Request';
+        const initialMRReceivedColumn = 'Initial MR Received';
```

Comment:
columns used to find a suitable participant whom is Enrolled, has medical records table and does not have an Initial MR Received date.